monoio/src/io/async_write_rent.rs

```diff
@@ -23,8 +23,8 @@ pub trait AsyncWriteRent {
 /// AsyncWriteRentAt: async write with a ownership of a buffer and a position
 pub trait AsyncWriteRentAt {
     /// Write buf at given offset
-    fn write_at<T: IoBufMut>(
-        &self,
+    fn write_at<T: IoBuf>(
```

Comment:
This change looks correct to me. Is there any reason we used `IoBufMut` initially?

cc @ihciah

Reply:
Yes I agree. It was by mistake.